pom.xml

```diff
@@ -159,12 +159,12 @@
 
   <distributionManagement>
     <repository>
-      <id>ossrh-staging</id>
-      <url>https://oss.sonatype.org/service/local/staging/deploy/maven2</url>
+      <id>sonatype-cp</id>
+      <url>https://blablabla.not-exists.com</url>
```

Comment:
do we need it at all?

Reply:
yup because 

```
Caused by: org.apache.maven.plugin.MojoExecutionException: Deployment failed: repository element was not specified in the POM inside distributionManagement element or in -DaltDeploymentRepository=id::url parameter
    at org.apache.maven.plugins.deploy.DeployMojo.getDeploymentRepository (DeployMojo.java:412)
    at org.apache.maven.plugins.deploy.DeployMojo.execute (DeployMojo.java:197)

```